src/Chainweb/BlockHeader.hs

```diff
@@ -745,6 +806,13 @@ timeBetween after before = f after - f before
     f :: BlockCreationTime -> Micros
     f (BlockCreationTime (Time (TimeSpan ts))) = ts
 
+-- | Absolute BlockHeight Difference
+--
+absBlockHeightDiff :: BlockHeader -> BlockHeader -> BlockHeight
+absBlockHeightDiff a b
```

Comment:
```suggestion
absBlockHeightDiff a b = abs $ _blockHeight a - _blockHeight b
```
unless I'm missing something?

Reply:
`BlockHeight` is a newtype wrapper around an unsigned type. Haskell uses "unsafe" C semantics for primitive types:

```
λ> abs $ BlockHeight 1 - BlockHeight 2
18446744073709551615
λ> abs $ BlockHeight 2 - BlockHeight 1
1
```